### ISSUE TITLE
fatal error after update

### ISSUE THREAD

**Issue by emergingdzns** [JosephSilber/bouncer#39]
I've done the update to Laravel 5.2 which was not easy. But now I get the following error:
```
PHP Fatal error:  Using $this when not in object context in /Users/emergingdzns/Cloud Drive/Sites/Symplur/signals/vendor/silber/bouncer/src/Database/Models.php on line 64
```
Fortunately I did this on a test branch though because we can't load the site at all now.

I'm not sure what could cause this conflict. Any ideas? Any particular code you'd like me to share?

**Comment by Gummibeer** [comment]
The code where you call `\Bouncer::setTable()` could be helpful.

**Comment by Arcesilas** [comment]
Related to #38 
You may change PHP configuration to hide the message (which is not recommanded), or use the fix by @marcusmoore88 , or wait for PR to be merged.

**Comment by JosephSilber** [comment]
Fixed in #38
